testing/py3-cairosvg/APKBUILD

```diff
@@ -0,0 +1,34 @@
+# Contributor: William Pitcock <nenolod@dereferenced.org>
+# Contributor: Stefan Wagner <stw@bit-strickerei.de>
+# Maintainer: William Pitcock <nenolod@dereferenced.org>
+pkgname=py3-cairosvg
+_pkgname=CairoSVG
+pkgver=2.2.1
+pkgrel=0
+pkgdesc="Cairo-based SVG renderer for python"
+url="http://cairosvg.org"
+arch="noarch"
+license="LGPL-3.0-or-later"
+options="!check"
```

Comment:
needs to explain why `check()` disabled

Reply:
I should have marked this as a WIP, I'm still working on getting checks right (missing dependencies).